src/validator.js

```diff
@@ -84,6 +100,59 @@ export default class Validator {
     return JSON.stringify.apply(null, args);
   }
 
+  textOutput() {
+    var out = [];
+
+    out.push(_('Validation Summary:'));
+    out.push('');
+    out.push(columnify(this.output.summary, {
+      showHeaders: false,
+      minWidth: 15,
+    }));
+    out.push('');
+
+    for (let type of constants.MESSAGE_TYPES) {
+      /*eslint-disable no-loop-func */
```

Comment:
Yeah it's the callback funcs passed into then etc. I reckon you're right, we should probably just turn that rule off.